reference/providers/kubernetes-v2/index.md

```diff
@@ -7,106 +7,83 @@ sidebar:
 
 ## No Restrictive Naming Policies
 
-In order to allow users to deploy existing manifests without rewriting them to
-adhere to the [Frigga naming convention](https://github.com/Netflix/frigga),
-resources relationships (between applications/clusters) are managed
-using [Kubernetes
-annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).
-These are managed using Spinnaker's
-[Moniker](https://github.com/spinnaker/moniker) library, and the
-policies/strategies are configurable per-account. See the below [reserved
-annotations](#reserved-annotations) section for more detail.
+You can deploy existing manifests without rewriting them to adhere to [Frigga](https://github.com/Netflix/frigga). Resource relationships (for example between applications and clusters) are managed using [Kubernetes annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/), and Spinnaker manages these using its [Moniker](https://github.com/spinnaker/moniker) library. 
+
+The policies and strategies are configurable per account. See [Reserved Annotations](#reserved-annotations) for more detail.
 
-## Accomodating Level-based Deployments
+## Accomodating Level-Based Deployments
 
 See the [Kubernetes API
 conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status)
-for a description of an edge- vs. level-based API.
-
-Prior providers in Spinnaker focused on tracking an operation that modified a
-cloud resource. For example, if a resize operation was run, Spinnaker
-would monitor that operation until the specific resize target was met. This
-runs counter to the Kubernetes level-based API, which only cares about the
-desired state of a resource being satisfied. In order to address this, we
-introduced the concept of __Manifest Stability__:
-
-A deployed manifest is consided __stable__ when the Kubernetes
-Controller-Manager no longer needs to modify it, and it is "ready". This
-definition is clearly very dependent on the manifest's kind (e.g. `Service` vs.
-`StatefulSet`), but for example, a `Deployment` is stable when its managed pods
-are updated, available, and ready (meaning they are running your desired
-container, and are serving traffic). Each kind's stability requirements are
-detailed in the [Resource Mapping](#resource-mapping) section.
-
-This concept of Manifest Stability is how Spinnaker ensures that operations
-have succeeded. Since there are a number of reasons why a manifest never
+for a description of edge-based vs. level-based APIs.
+
+Other providers in Spinnaker track operations that modify cloud resources. For example, if you run a resize operation, Spinnaker monitors that operation until the specified resize target is met. This is different from how the Kubernetes level-based API does  it: Kubernetes only tries to satisfy the desired state.
```

Comment:
This paragraph is a little confusing to me - it compares how Spinnaker monitors operations to how the Kubernetes API works, instead of how Spinnaker monitors operations for existing providers to operations in the v2 provider.